R/backend-redshift.R

```diff
@@ -57,7 +57,32 @@ sql_translation.RedshiftConnection <- function(con) {
       }
     ),
     sql_translator(.parent = postgres$aggregate),
-    sql_translator(.parent = postgres$window)
+    sql_translator(.parent = postgres$window,
+      # https://docs.aws.amazon.com/redshift/latest/dg/r_WF_LAG.html
+      lag = function(x, n = 1L, default = NA, order_by = NULL) {
```

Comment:
I think this would be simpler if you just omitted the `default` argument all together — then the user will just get the regular `unused argument (default = 1)` error